src/components/api-reference/schemas/measurement-sets.js

```diff
@@ -12,9 +12,11 @@ const FIELDS = [
   {name: 'submissionId', value: 'string', description: 'The id of the submission in which the measurement set belongs.'},
   {name: 'category', value: 'string', description: 'The category of the measurement set. Acceptable values are <b>"ia"</b>, <b>"aci"</b> and <b>"quality"</b>.', notes: 'writable'},
   {name: 'submissionMethod', value: 'string', description: 'The method by which the measurement set data was submitted. Acceptable values are <b>"cmsWebInterface"</b>, <b>"electronicHealthRecord"</b>, <b>"claims"</b>, <b>"registry"</b>, <b>"certifiedSurveyVendor"</b> and <b>"administrativeClaims"</b>.', notes: 'writable'},
+  {name: 'programName', value: 'string', description: 'The quality payment program under which the measurement set belongs, MIPS or CPC+. Acceptable values are <b>"mips"</b> and <b>"cpcPlus"</b>. If not provided, programName will be recorded as "mips".', notes: 'writable, optional'},
+  {name: 'practiceId', value: 'string', description: 'The ID of the practice associated with the measurement set. Required if programName is set to "cpcPlus". Must be omitted if programName is not set to "cpcPlus".', notes: 'writable, optional'},
   {name: 'performanceStart', value: 'string', description: 'A date in RFC 3339 format with only the date part (for instance, "2013-01-15"). The first date when the measurement data is applicable.', notes: 'writable'},
   {name: 'performanceEnd', value: 'string', description: 'A date in RFC 3339 format with only the date part (for instance, "2013-01-15"). The last date when the measurement data is applicable.', notes: 'writable'},
-  {name: 'measureSet', value: 'string', description: 'The speciality set which the measurement set is attesting. Acceptable values are "generalSurgery" or "other"', notes: 'writable, optional'},
+  {name: 'measureSet', value: 'string', description: 'The speciality set which the measurement set is attesting. Acceptable values are <b>"generalSurgery"</b> or <b>"other"</b>', notes: 'writable, optional'},
```

Comment:
Out of scope of this ticket but this is outdated...we accept a whole slew of measure sets now!

Reply:
wow yeah! I can add them into this pr, seems reasonable.

allergyImmunology, anesthesiology, cardiology, dermatology, diagnosticRadiology, electrophysiologyCardiacSpecialist, emergencyMedicine, gastroenterology, generalOncology, generalPracticeFamilyMedicine, generalSurgery, hospitalists, internalMedicine, interventionalRadiology, mentalBehavioralHealth, neurology, obstetricsGynecology, ophthalmology, orthopedicSurgery, otolaryngology, pathology, pediatrics, physicalMedicine, plasticSurgery, preventiveMedicine, radiationOncology, rheumatology, thoracicSurgery, urology, vascularSurgery

Reply:
@marimiyachi can you just confirm "other" is no longer an option?

Reply:
Yup - other is not an option.

Reply:
thanks, added